src/components/frame/SubscribedColoniesList/SubscribedColoniesList.tsx

```diff
@@ -20,17 +20,10 @@ const MSG = defineMessages({
 });
 
 const SubscribedColoniesList = () => {
-  const { user, userLoading, updateUser } = useAppContext();
+  const { user, userLoading } = useAppContext();
   const canInteractWithNetwork = useCanInteractWithNetwork();
   const isMobile = useMobile();
 
```

Comment:
Removed as of this comment as it is now redundant - https://github.com/JoinColony/colonyCDapp/pull/83/files#r1029539075